### ISSUE TITLE
WP QuickLaTeX images are being wrapped in no-caption tags

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#2213]
We recently improved a regular expression that wraps images in no-caption classes: https://github.com/pressbooks/pressbooks/pull/2173. However, this fix causes problems for images created with WP QuickLaTeX, since math expressions that were previously displayed inline are now being wrapped in no-caption divs and centered aligned on their own line. We should adjust the regular expression so that it does not accidentally catch these images. Our MathJax generated images in EPUB and PDF exports are targeted accidentally as well.

Basically, the problem is that users can no longer insert inline images, since these are now wrapped in div tags and centered on their own line. See an example at https://viva.pressbooks.pub/openmusictheory/chapter/simple-meter-and-time-signatures/

![Screenshot from 2021-06-17 07-23-23](https://user-images.githubusercontent.com/13485451/122415673-f926f780-cf3c-11eb-92b2-3f0b3746190d.png)

Related issue that also needs to be addressed: https://github.com/pressbooks/pressbooks/issues/2218
